Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -36,20 +38,22 @@ public static void TearDownContext()
     }
 
     /*
-     * This is the base class for all tests that call async metodes in the library - it does not use a special SynchronizationContext
+     * This is the base class for all tests that call async methods in the library - it does not use a special SynchronizationContext
      * 
      */
-    [SetUpFixture]
+    [TestFixture]
     public class SFBaseTestAsync
     {
-        private const string connectionStringWithoutAuthFmt = "scheme={0};host={1};port={2};" +
-            "account={3};role={4};db={5};schema={6};warehouse={7}";
+        private const string ConnectionStringWithoutAuthFmt = "scheme={0};host={1};port={2};" +
+                                                              "account={3};role={4};db={5};schema={6};warehouse={7}";
+        private const string ConnectionStringSnowflakeAuthFmt = ";user={0};password={1};";

```

Comment:
Since we are touching this there are multiple ConnectionsStrings in the code base:
- without auth but later on added with user password
- with user and pwd
- with jwt authenticator
- with external browser auth
- with application name
If that's not much of a work why don't we move all of those here and reuse them in the tests.
Some special case for just one test I would leave as is.

Reply:
I don't think we should include this in this PR. And I'm not sure if I agree. This are specific auths and requires specific cases. I wouldn't pull everything to the uber base class. Let's keep only common auth data here?

Reply:
I agree with Piotr that we should keep the common data here as it is.